st2actions/st2actions/workflows/workflows.py

```diff
@@ -203,6 +211,13 @@ def handle_action_execution(self, ac_ex_db):
         # Process completion of the action execution.
         wf_svc.handle_action_execution_completion(ac_ex_db)
 
+    def _is_orquesta_execution(self, ac_ex_db):
+        """
+        Return True if particular execution object represents an orquesta workflow, False
```

Comment:
This description is wrong. When "orquesta" is in the context, this means the action execution belongs to a task in an orquesta workflow. This doesn't mean the action execution is an orquesta workflow.

Reply:
Fixed.